### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by FennyWilriani** [lab-brussels-1/HYF-Redesign-Group-3#2] [Announcements]
Review works

**Comment by sammou00** [comment]
Hello, 
Very good user-personas.  by the way HYF, is not only teaching front-end also back-end, on the back-end we used `node.js` But, I have no idea about your current class. 

- very strange class names 
- 
![image](https://user-images.githubusercontent.com/67711948/163689754-53758433-0430-4279-bbb4-38dea9934810.png)

![image](https://user-images.githubusercontent.com/67711948/163689783-b4ecc6e2-4bbf-45b0-893b-67124c95bdbb.png)

-  it is important to mention the versions as well

Good job

**Reply by FennyWilriani** [reply]
Apologies for finding your comment after project review @samirm00 .. and thank you so much for your feedback, I will bring up this naming convention issue to the team meeting before we continue with our project.

**Reply by sammou00** [reply]
Hello, don't worry everything is good.
